src/codegen/riscv64/assembler-riscv64-inl.h

```diff
@@ -243,7 +243,10 @@ void Assembler::emit(ShortInstr x) {
   if (!is_buffer_growth_blocked()) {
     CheckBuffer();
   }
+  DEBUG_PRINTF("%p: ", pc_);
+  disassembleInstr(x);
   EmitHelper(x);
+  CheckTrampolinePoolQuick();
```

Comment:
Why does this emit call `CheckTrampolinePoolQuick` but the others do not?

Reply:
@luyahan would you be able to answer this? I'm not too sure.

Reply:
@luyahan, can you answer this question? The other comments for this PR are resolved.

Reply:
It is similiar with this:
```
void Assembler::emit(Instr x) {
  if (!is_buffer_growth_blocked()) {
    CheckBuffer();
  }
  DEBUG_PRINTF("%p: ", pc_);
  disassembleInstr(x);
  EmitHelper(x);
  CheckTrampolinePoolQuick();
}
```

Reply:
Sorry, you're right. I must've looked at this wrong before.